ecommerce/extensions/catalogue/tests/test_migrate_course.py

```
@@ -303,8 +302,22 @@ def test_handle_with_commit(self):
             # Verify that the migrated course was published back to the LMS
             self.assertTrue(mock_publish.called)
 
+    def test_handle_with_commit(self):
+        """ Verify the management command retrieves data, and saves it to the database. """
+        self._handle_with_commit()
         self.assert_course_migrated()
 
+    def test_course_migrated_without_stale_modes(self):
+        """
+        Verify the course was migrated without stale modes.
+        """
+        self._handle_with_commit()
```

This should raise an error instead of simply dropping the product.

It's also not clear why this was broken out into a separate method, given that the data has not changed between the two tests.